tests/data/native_encoding/generate_fixtures.patch

```diff
@@ -0,0 +1,160 @@
+diff --git a/lib/value/src/value/arbitrary.rs b/lib/value/src/value/arbitrary.rs
```

Comment:
This patch will probably not age too well, but I appreciate a lot the context you gave in the README 👍

Reply:
Yes, I figured it wouldn't be applicable for long but was easier to include than list in depth what changes I had to make 😄